js/ui/cardList.js

```diff
@@ -1,11 +1,25 @@
 import { getCardComponent } from './cardComponent.js';
+import { getCardEdit } from './cardEdit.js';
+import { getCardEditComponent } from './cardEditComponent.js';
 
 export const getCardList = (params) => {
   const cardList = document.getElementById('card-list');
   while (cardList.firstChild) cardList.firstChild.remove();
 
-  params.flashcards.forEach((flashcard) => {
+  params.flashcards.forEach((flashcard, index) => {
+    let isFront = true;
     const card = getCardComponent(flashcard);
+    card.id = index;
+    const editCardIcon = card.querySelector('.edit-icon');
+    editCardIcon.addEventListener('click', function () {
```

Comment:
**nitpick:** I wonder why this logic is a part of list - I mean clicking on edit card button should be a part of one of your card components